README.md

```diff
@@ -55,7 +60,7 @@ Ursa uses the following external dependencies:
 Ursa uses the following external dependencies for testing:
 
 - [libsodium
-  1.0.18](https://download.libsodium.org/libsodium/releases/old/libsodium-1.0.16.tar.gz)
+  1.0.18](https://download.libsodium.org/libsodium/releases/libsodium-1.0.16.tar.gz)
```

Comment:
Hi - the text says 1.0.18, the URL says 1.0.16. Which is correct?

Reply:
Ha, someone had fixed this problem, so this issue should be closed